cli/root.go

```diff
@@ -15,10 +15,12 @@ import (
 
 var rootRepositoryFlag string
 var rootFormatFlag string
+var rootVersionPrefixFlag string
 
 func init() {
 	RootCommand.Flags().StringVarP(&rootRepositoryFlag, "repository", "r", ".", "sets the path to the repository")
 	RootCommand.Flags().StringVarP(&rootFormatFlag, "format", "f", "version", "sets the output format")
+	RootCommand.Flags().StringVarP(&rootVersionPrefixFlag, "version-prefix", "p", "", "sets the version prefix")
```

Comment:
Although we are going to rename the variable, we should not change the description of the help message, so please keep it as it is 👍